src/actions/actionUpdateUserProfile.ts

```diff
@@ -141,6 +143,16 @@ async function _actionUpdateUserProfile(data: z.infer<typeof zodUpdateUserProfil
         phoneNumber,
       },
     })
+
+    const analytics = getServerAnalytics({
+      localUser,
+      userId: authUser.userId,
+    })
+    await analytics
+      .track('User SMS Opt-In', {
+        provider: 'twilio',
+      })
+      .flush()
```

Comment:
np: if you're doing only one track you can condense this into one statement
```suggestion
    await getServerAnalytics({
      localUser,
      userId: authUser.userId,
    })
      .track('User SMS Opt-In', {
        provider: 'twilio',
      })
      .flush()
```